metadata-ingestion/source_docs/bigquery.md

```diff
@@ -0,0 +1,111 @@
+| `table_pattern.deny`   |          |              | Regex pattern for tables to exclude from ingestion.                                                                                                                                     |
+| `schema_pattern.allow` |          |              | Regex pattern for schemas to include in ingestion.                                                                                                                                      |
+| `schema_pattern.deny`  |          |              | Regex pattern for schemas to exclude from ingestion.                                                                                                                                    |
+| `view_pattern.allow`   |          |              | Regex pattern for views to include in ingestion.                                                                                                                                        |
+| `view_pattern.deny`    |          |              | Regex pattern for views to exclude from ingestion.                                                                                                                                      |
+| `include_tables`       |          | `True`       | Whether tables should be ingested.                                                                                                                                                      |
+| `include_views`        |          | `True`       | Whether views should be ingested.                                                                                                                                                       |
+
+# BigQuery Usage Stats
+
+## Setup
+
+To install this plugin, run `pip install 'acryl-datahub[bigquery-usage]'`.
+
+## Capabilities
+
+This plugin extracts the following:
+
+- Statistics on queries issued and tables and columns accessed (excludes views)
+- Aggregation of these statistics into buckets, by day or hour granularity
+
+Note: the client must have one of the following OAuth scopes, and should be authorized on all projects you'd like to ingest usage stats from.
+
+- https://www.googleapis.com/auth/logging.read
+- https://www.googleapis.com/auth/logging.admin
+- https://www.googleapis.com/auth/cloud-platform.read-only
+- https://www.googleapis.com/auth/cloud-platform
+
+:::note
+
+This source only does usage statistics. To get the tables, views, and schemas in your BigQuery project, use the `bigquery` source described above.
+
+:::
+
+## Quickstart recipe
+
+Check out the following recipe to get started with ingestion! See [below](#config-details) for full configuration options.
+
+```yml
+source:
+  type: bigquery-usage
+  config:
+    projects:
+      - project_id_1
+      - project_id_2
+
+    bucket_duration: "DAY"
```

Comment:
What's the default lookback window - ie how far back will the usage stats look? Just yesterday by default I believe. On initial run, we may want a longer lookback window. Something we will likely want to add (maybe not in this PR)